### ISSUE TITLE
download.vscodium.com not working

### ISSUE THREAD

**Issue by locture** [VSCodium/vscodium#1094]
Currently there is a certification error blocking the users from using the download.vscodium.com mirror (which was documented on the download page).

On the other hand, the paulcarroty‘s repo and vscodium.c7.ee mirror both works fine, so this should be an issue only with the official CDN mirror.

Regards,

Loc

**Comment by aphive** [comment]
Is that even a site, not sure where you got that url from ... 
Why not just download from the releases page as the download link on vscodium.com sends you?
https://github.com/VSCodium/vscodium/releases

**Comment by locture** [comment]
> Is that even an official part of the site, not sure where you got that url from ... Why not just download from the releases page as the official download link on vscodium.com sends you? https://github.com/VSCodium/vscodium/releases

Yes it is. It is listed as the CDN on the official website https://vscodium.com/. Check the section on the Install with Package Manager (Linux) section.

[@paulcarroty](https://github.com/paulcarroty) has set up a [repository](https://gitlab.com/paulcarroty/vscodium-deb-rpm-repo) for VSCodium. The instructions below are adapted from there with [CDN](https://download.vscodium.com/) mirror. Any issues installing VSCodium using your package manager should be directed to that repository’s issue tracker.

It is also listed as it is on Paulcarroty's repository.

Why dont I just use the rpm from the github download? Because I am using it under Linux and I would prefer installing it using my distro's package manager and get better update management.
